Zotero/Scenes/AppCoordinator.swift

```diff
@@ -250,18 +241,19 @@ extension AppCoordinator: AppDelegateCoordinatorDelegate {
                         }
                         return presentation
 
-                        func loadPDFPresentation(key: String, libraryId: LibraryIdentifier, coordinator: DbCoordinator) throws -> ItemPresentation? {
+                        func loadItemPresentation(kind: OpenItem.Kind, coordinator: DbCoordinator) throws -> ItemPresentation? {
```

Comment:
This seems to do mostly the same work that `CustomURLController` does in `func openItem(path: String, queryItems: [URLQueryItem]) -> Kind?`. Could it be unified?